management/src/main/java/nearsoft/skytouch/management/service/Impl/ProductServiceImpl.java

```diff
@@ -0,0 +1,23 @@
+package nearsoft.skytouch.management.service.Impl;
+
+import nearsoft.skytouch.common.model.Product;
+import nearsoft.skytouch.management.repository.ProductRepository;
+import nearsoft.skytouch.management.service.ProductService;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+@Service
+public class ProductServiceImpl implements ProductService {
```

Comment:
I like the idea about using interfaces and implementations, nice approach. However, you can make this class package-private, and you could make use of it by its interface.

Reply:
Ok, mm i don't understand enough why use a package-private class in this case, can you explain me why?

Reply:
ok got it, i'll change that, thanks!